pylti1p3/lineitem.py

```diff
@@ -215,7 +217,7 @@ def set_submission_review(
 
         return self
     
-    def get_submission_type(self) -> t.Optional[TSubmissionType]:
+    def get_submission_type(self) -> Union[TSubmissionType, None, _MISSING_TYPE]:
```

Comment:
Since we are pyton 3.12+, can't we use the nicer typing format?

```suggestion
    def get_submission_type(self) -> TSubmissionType | None | _MISSING_TYPE:
```